src/marqo/tensor_search/validation.py

```diff
@@ -556,3 +558,15 @@ def validate_multimodal_combination_mapping(field_mapping: dict):
                 f"In multimodal_combination fields, weight must be an int or float."
                 f"Please check `https://docs.marqo.ai/0.0.15/Advanced-Usage/document_fields/#multimodal-combination-object` for more info."
             )
+
+
+def validate_score_modifiers_object(score_modifiers: List[dict]):
+    try:
+        jsonschema.validate(instance=score_modifiers, schema=score_modifiers_object_schema)
+        return score_modifiers
+    except jsonschema.ValidationError as e:
+        raise InvalidArgError(
+            f"Error validating custom_score_fields object = `{object}`. Reason: \n{str(e)} "
```

Comment:
Where does `object` get declared, it would be important to test this

Reply:
updated